src/D2L.CodeStyle.Analyzers/UnsafeStaticsAnalyzer.cs

```diff
@@ -98,6 +98,21 @@ public sealed class UnsafeStaticsAnalyzer : DiagnosticAnalyzer {
                     var diagnostic = Diagnostic.Create( Rule, variable.GetLocation(), variable.Identifier.ValueText, symbol.Type.GetFullTypeName() );
                     context.ReportDiagnostic( diagnostic );
                 }
+
+                if( symbol.Type.IsImmutableCollectionType() ) {

```

Comment:
I don't, because a non-immutable collection is flagged no matter what the element type is. 

Essentially the error message should look like this:
`List<foo>` --> List is mutable (don't bother even checking the element type)
`ImmutableList<foo>` --> foo is mutable
`ImmutableList<DateTime>` -- safe